source/blog/_posts/2016-02-11-hackathon-2016-1.md

```diff
@@ -0,0 +1,266 @@
+support-department to get more information about the errors that occur in a shop.
+The plugin saves all occurring errors with all the information we can get - the user, that got the error, which
+articles were in the basket at that moment, on which page (controller / action) he got that page and all the information
+that have been saved to the session. Those information can be read in the backend then. 
+If it's a known error to us, there might also be a suggestion on how to fix that special issue or how that issue may occur.
+
+Additionally the plugin should provide some helpful functions, e.g. deactivating all 3rd-party plugins and re-activating
+them later or simulating the error, that just occurred.
+
+<div style="clear:both"></div>
+	
+## Syncing down the house		
+
+**Team**: Jan Philipp Pietzrzyk<br>
+**Project**: Shopware peer to peer network
+
+<img src="/blog/img/hackathon-2016-1/p2p.png" class="is-float-left" style="width:300px" />
+BitTorrent, Napster, Morpheus all these illegal file sharing clients share a underused and very interesting network architecture:
+Peer to Peer Networking. In my project I connected multiple Shopware shops, that act like a bee hive and communicate
+and synchronize with each other. This is done without a central server but by propagating data through a number of hops
+or peers. So although no shop by itself knows everyone else in the network, eventually all the data syncs up and every
+store has the exact same data set.
+
+To see this in action I created a minimal website, reminiscent of the webs past [remark DN: framesets FTW!]. But I hope you can imagine
+that even though there is no visual style to it, it makes a lot of fun seeing a simple change in a product propagating 
+through a whole network of shops.
+
+
+<div style="clear:both"></div>
+
+## Shopware Global Media		
+
+**Team**: Dennis Mader, Arnold Stoba<br>
+**Project**: Central asset managment
+
+Due to the fact that the shopware universe is constantly evolving, we decided to create a solution to host global media
+assets in a cloud environment like Amazon S3. At the moment all website assets are located on their specific server
+which creates tons of junk media with no possibility to manage these globally. With this attempt we are able to make
+sure that all relevant shopware media assets (for example our logo, colors, fonts) have the exact same version on every
+shopware site. The new assets.shopware.com domain. which points to an asset-specific amazon s3 bucket, can now be
+implemented as a CDN-like provider on all pages of the shopware universe.	
+
+<div style="clear:both"></div>
+
+## Light it up			
+
+**Team**: Stephan Pohl<br>
+**Project**: Continuous integration visualisation   
+
+Continuous integration is a key factor for high quality software. It ensures a continuous quality throughout the software.
+On the other hand developers have to be aware of the current status of the project in a non-disturbing way matter.
```

Comment:
@dnoegel You can remove the "matter" e.g.

"On the other hand developers have to be aware of the current status of the project in a non-disturbing way ~~matter~~".